terratest/utils/extensions.go

```diff
@@ -211,6 +196,22 @@ func (w *Workflow) Start() (*Instance, error) {
 		k8s.WaitUntilServiceAvailable(w.t, w.k8sOptions, w.state.testApp.name, 60, 1*time.Second)
 		w.state.testApp.isRunning = true
 	}
+
+	// gslb
+	if w.settings.gslbResourcePath != "" {
+		w.t.Logf("Create ingress %s from %s", w.state.gslb.name, w.settings.gslbResourcePath)
+		k8s.KubectlApply(w.t, w.k8sOptions, w.settings.gslbResourcePath)
+		k8s.WaitUntilIngressAvailable(w.t, w.k8sOptions, w.state.gslb.name, 60, 1*time.Second)
+		ingress := k8s.GetIngress(w.t, w.k8sOptions, w.state.gslb.name)
+		require.Equal(w.t, ingress.Name, w.state.gslb.name)
+		w.settings.ingressName = w.state.gslb.name
+	}
+
+	// ingress
+	if w.settings.ingressResourcePath != "" {
```

Comment:
is this intentional "NOOP" or am I missing something?

Reply:
Dunno, it was there so I'm leaving it as it is. Maybe @kuritka have an idea?

Reply:
Yep, good to check. To me, this looks like a dead code.
Also surprised it got skipped the linters somehow, staticcheck would definitely treat this as an empty branch:
https://staticcheck.io/docs/checks/#SA9003